examples/comb/ex_comb.cpp

```diff
@@ -0,0 +1,54 @@
+#include "daisysp.h"
+#include "daisy_seed.h"
+
+using namespace daisysp;
+using namespace daisy;
+
+static DaisySeed seed;
+Comb flt;
+static WhiteNoise noise;
+
+static void AudioCallback(float *in, float *out, size_t size)
+{
+	float output;
+    for (size_t i = 0; i < size; i += 2)
+    {
+        output = noise.Process();
+	    
+        output = 0.5 * flt.Process(output);
+
+    	// left out
+        out[i] = output;
+
+        // right out
+        out[i+1] = output;
+    }
+}
+
+int main(void)
+{
+	// initialize seed hardware and daisysp modules
+        float sample_rate;
+	seed.Configure();
+	seed.Init();
+	sample_rate = seed.AudioSampleRate();
+	
+	float buf[9600];
```

Comment:
I would personally declare my buffers in the global space with my objects. So that I can move them to other memory segments if I want to. (and so the buffer can be grouped with the object it goes with). 

BUT this is should work fine as main() won't ever return.